Corona-Warn-App/src/main/java/de/rki/coronawarnapp/datadonation/survey/Surveys.kt

```diff
@@ -49,7 +50,9 @@ class Surveys @Inject constructor(
     }
 
     suspend fun resetSurvey(type: Type) {
-        // TODO
+        if (type == Type.HIGH_RISK_ENCOUNTER) {
```

Comment:
Minor: please add a Timber to the `if` and `else` case

Reply:
Moved the log statement from `RiskLevelChangeDetector` to this if.